packages/node-runtime-worker-thread/src/rpc.ts

```diff
@@ -98,7 +102,17 @@ export type WithClose<T> = { [k in keyof T]: T[k] & { close(): void } };
 
 export function exposeAll<O>(obj: O, messageBus: RPCMessageBus): WithClose<O> {
   Object.entries(obj).forEach(([key, val]) => {
-    const { close } = expose(key, val, getRPCOptions(messageBus));
+    const { close } = expose(
+      key,
+      async(...args: unknown[]) => {
+        try {
+          return await val(...args);
+        } catch (e) {
+          return { [ERROR]: serializeError(e) };
```

Comment:
Just an idea, rather than having this special property to guess that something is an error i think is better to have these events / messages to take a slightly more robust form. Similarly to the serialization of the results you could have a `type` and a `payload` properties:

```
{type: 'message', payload: await val(...args)}
{type: 'error', payload: serializeError(e)}
```

Reply:
Yeah, makes sense! Will change

Reply:
Done in 6cd0e3b